src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -1129,6 +1129,10 @@ namespace OpenMS
       }
       chromatogram.setMetaValue("product_mz", transition->getProductMZ());
       chromatogram.setMetaValue("precursor_mz", transition->getPrecursorMZ());
+      Precursor prec; prec.setPosition(transition->getPrecursorMZ());
```

Comment:
I wonder if that breaks the SWATH tests. But It looks like the metaValues are legacy anyways. Some debug messages indirectly depend on the Precursor being set.

Reply:
Ok have to check if the failing tests are just cosmetic.

Reply:
@cbielow @hroest Any comments on this. Feels like it should be okay. Tests are running. Are the Precursor and Product fields the "new" alternative to the metavalues?